muni_portal/settings/base.py

```diff
@@ -286,3 +286,4 @@
 
 # https://github.com/web-push-libs/pywebpush
 VAPID_PRIVATE_KEY = env.str("VAPID_PRIVATE_KEY", "vapid_private_key.pem")
+VAPID_PUBLIC_KEY = env.str("VAPID_PUBLIC_KEY", "vapid_public_key.pem")
```

Comment:
Please change these not to have a default value, so that the server fails to start if they are not configured. Then provide a default value in docker-compose.yml

Reply:
both public and private